habushu-maven-plugin/pom.xml

```diff
@@ -191,6 +209,9 @@
                             cucumber.junit-platform.naming-strategy=long
                         </configurationParameters>
                     </properties>
+                    <excludes>
```

Comment:
S: I believe if you rename this class to not end with `TestCase`, you can eliminate this.  To me, that's probably worth the simplification.

Reply:
could not figure out for the life of me why this was getting triggered. that worked, thank you!